boxing/src/gym/Boxer.java

```diff
@@ -0,0 +1,72 @@
+package gym;
+
+import java.util.Objects;
+
+public class Boxer{
+    private String name;
+    private int age;
+    private int hitPoints;
+    private int power;
+    private Integer wins;
+
+    public Boxer() {
+
+    }
+
+    public Boxer(String name, int age, int hitPoints, int power) {
+        this.name = name;
+        this.age = age;
+        this.hitPoints = hitPoints;
+        this.power = power;
+        wins = 0;
+    }
+
+    public Integer getWins() {
+        return wins;
+    }
+
+    public void setWins() {
+        this.wins++;
+    }
```

Comment:
Setters usually should work in a slightly different way - you pass the parameter and you set that value. This solution works, but is kind of rigid.